task/gcs-generic/0.1/gcs-generic.yaml

```diff
@@ -26,11 +26,11 @@ spec:
    description: |
       The path inside the credentials workspace to the GOOGLE_APPLICATION_CREDENTIALS key file.
    type: string
-   default: service_account.json
+   default: ''
  - name: image
    description: google cloud image that will be used in steps.
    type: string
-   default: google/cloud-sdk
+   default: gcr.io/google.com/cloudsdktool/cloud-sdk
```

Comment:
can you please also add a tag or a digest for the image? :slightly_smiling_face:

Reply:
This is intentional - this pulls in the latest version by default, and users can pin to specific versions by setting the image param.

Set the tag to `slim`, but this is to pull in a minimized version, not for any version pinning.

Reply:
my reason for specifying the tag was that in future we are planning to add catlin(catalog linter) to catalog CI which has a requirement that the image should be tagged

Reply:
I think from a security point of view we don't want to have by default pulling the `latest`, isnt it ? cc @dlorenc